web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -275,6 +293,22 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
     handleConfirmClose();
   }, [csrf, periods, dispatch, toDelete, handleConfirmClose]);
 
+  const updateReviewPeriodDates = useCallback(
+    async period => {
+      if (!csrf) {
+        return;
+      }
+      const res = await updateReviewPeriod(period, csrf);
+      const data = res?.payload?.data ?? null;
+      if (data) {
+        dispatch({ type: UPDATE_REVIEW_PERIODS, payload: [...periods] });
+      } else {
+        console.log(res?.error);
```

Comment:
I imagine we need better error handling here like display a message in a dialog.
I'm not familiar with how other similar errors are handled in Check-ins.

Reply:
Adjusted with toast

Reply:
Nice! If you also want to keep the console output, I would change it to use `console.error` instead of `console.log`. That will make it appear in red.